nomad/structs/structs.go

```diff
@@ -4527,6 +4529,31 @@ type JobTaggedVersion struct {
 	TaggedTime  int64
 }
 
+type JobApplyTagRequest struct {
+	JobID string
+	Name  string
+	// Version     string
+	Description string
+	Version     *uint64
+	// Tag     *JobTaggedVersion
+	// QueryOptions
+	WriteRequest
+}
+
+type JobUnsetTagRequest struct {
+	JobID string
+	Name  string
+	// QueryOptions
+	WriteRequest // TODO: do deletes get a WriteRequest?
+}
```

Comment:
Yes, deletes get a `WriteRequest`. Also, do we really need a separate RPC message for this? It seems like we could untag a version using the same RPC as above with a little cleverness with the fields.

Reply:
Removed the second RPC message!